airflow/www/views.py

```diff
@@ -3194,16 +3194,22 @@ def action_has_dag_edit_access(action_func: Callable) -> Callable:
     @wraps(action_func)
     def check_dag_edit_acl_for_actions(
         self,
-        items: Optional[Union[List[TaskInstance], List[DagRun], TaskInstance, DagRun, None]],
+        items: Optional[Union[Iterable[TaskInstance], Iterable[DagRun], TaskInstance, DagRun]],
         *args,
         **kwargs,
-    ) -> None:
+    ):
```

Comment:
Why removing this?

Reply:
When rewriting the function I must have accidentally removed it. Adding it back 👍